alfresco-actuators/src/main/resources/alfresco/module/alfresco-actuators/log4j.properties

```diff
@@ -0,0 +1 @@
+log4j.logger.eu.xenit=INFO
```

Comment:
This logger is a bit too broad to actually ship with this module, I think

Reply:
Reduced scope.